utils/gtrnadb.py

```diff
@@ -42,6 +46,42 @@ def setup():
                     get_trnascan_cm(domain, isotype)
 
 
+def verify_anticodon(isotype, anticodon, start, end):
+    """
+    When multiple models are possible, select the most likely one.
+
+    Currently only Cys, Leu, and Ser TRNAinf-mito-vert have more than one CM/template.
+
+    Leu
+    LeuTAA and LeuTAG do not have any structural or length differences.
+    Either template can be used.
+
+    Cys
+    There is only one anticodon for Cys
+    <= 61 bp without D-arm.
+    >= 65 with D-arm (typical Cys model).
+    in between can be either.
+
+    Ser
+    >= 70 bp are usually SerTGA (with a D-arm).
+    < 65 are usually SerGCT (without a D-arm).
+    in between can be either.
+    """
+    if anticodon != "NNN":
+        return anticodon
+    seq_length = max(start, end) - min(start, end) + 1
```

Comment:
minor: `abs(end - start) + 1` does the same and is a bit shorter

Reply:
Thank you very much. All suggested changes pushed now ✅